### PR TITLE
Tickets/dm 29420

### PR DESCRIPTION
**Pull request by wvreeven**

### CODE REVIEW

**Review by tribeiro — CHANGES_REQUESTED** [review]
Please, rebase to master instead of merging master to your branch.

**Comment by wvreeven** [comment]
Yes, silly of me. Done.

File: build/deploy-env/conda-aarch64/Dockerfile [hunk 1]
```diff
@@ -45,8 +45,6 @@ USER saluser
 WORKDIR ${WORKDIR}
 
 ENV numpy=1.20
-ENV ts_idl=2.3.0
```
Comment (tribeiro) [inline build/deploy-env/conda-aarch64/Dockerfile:48]: You need both ts_idl and ts_salobj (see line 59 and 60), but they must be `ARG` not `ENV`. You probably need to convert `ENV numpy=1.20` to `ARG numpy=1.20` as well.

**Review by tribeiro — CHANGES_REQUESTED** [review]

File: build/deploy-env/conda-aarch64/Dockerfile (hunk 1 above)
Comment (wvreeven) [reply build/deploy-env/conda-aarch64/Dockerfile:48]: Good catch. I totally overlooked that. Done.

**Review by tribeiro — APPROVED** [review]
Looks good! Thanks for the update

**Pull request merged**